include/users.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+    function getAllUsers() {
+        $users = dbQuery("
+            SELECT *
+            FROM users
```

Comment:
This is fine as a start, but if you have archived users, I imagine you may not want to select those?